addons/l10n_es_edi_sii/views/res_config_settings_views.xml

```diff
@@ -24,12 +24,28 @@
                                   groups="base.group_multi_company"/>
                             <div class="content-group">
                                 <div class="mt16">
+                                    <label for="l10n_es_edi_tax_agency"/>
                                     <field name="l10n_es_edi_tax_agency"/>
+                                    <div class="text-muted" attrs="{'invisible': [('l10n_es_edi_tax_agency', '!=', False)]}">
+                                        No tax agency selected: SII not activated.
+                                    </div>
+                                    <div class="text-muted" attrs="{'invisible': [('l10n_es_edi_tax_agency', '=', False)]}">
+                                        Tax agency selected: SII is activated.
+                                    </div>
                                     <br/>
-                                    Check this box if test env: <field name="l10n_es_edi_test_env"/>
-                                    <p attrs="{'invisible': [('l10n_es_edi_certificate_ids', '!=', [])]}">
-                                        Go to Configuration > Certificates [ES] to add your certificate.
-                                    </p>
+                                    <div class="o_row">
+                                        <label for="l10n_es_edi_test_env"/>
+                                        <field name="l10n_es_edi_test_env"/>
+                                    </div>
+                                    <div class="text-muted" attrs="{'invisible': [('l10n_es_edi_test_env', '=', False)]}">
+                                        Test mode: EDI data is sent to separate test servers.
+                                    </div>
+                                    <div class="text-muted" attrs="{'invisible': [('l10n_es_edi_test_env', '!=', False)]}">
+                                        Production mode: EDI data is sent to the official agency servers.
+                                    </div>
+                                    <br/>
+                                    Add your certificate:
```

Comment:
I guess this could be "Manage certificates" or even just "Certificates" or even just have the button with no text before.